### DISCUSSION TITLE
[FEATURE] 使用cdn情况，默认配置下，网站日志只获取cdnIP

### DISCUSSION THREAD

**Discussion by R0A1NG** [1Panel-dev/1Panel#5343] [Q&A]
### 1Panel 版本

v1.10.10-beta.2

### 请描述您的需求或者改进建议

![1717749906385](https://github.com/1Panel-dev/1Panel/assets/66672984/bbe7a271-bd3e-4d98-9346-3d6ac68e3023)

能否把waf获取真实IP的功能，给网站访问日志获取IP使用

**Comment by zhengkunwang223** [comment]
这个暂时没有方案，你可以通过修改 openresty 配置文件的方式来实现
